### PR TITLE
fix iOS video: don't use Hls()

### PR DESCRIPTION
**Pull request by elv-todd**
on iOS, use playerElement.src directly, don't assign a new Hls()

### CODE REVIEW

**Review by elv-peter — COMMENTED** [review]
Hmm, the logic probably should be `if player != bitmovin { all the other logic }`

**Comment by elv-todd** [comment]
IMG_2346

showing `examples/basic-video-example.html`
`hq__FRHfVK9ppcowDmB6s48cZtHFbJ5Qgduq126eLXJBfPbrmkATowwzz9mjzMfns5B5WpHwBfFYfG`
from `https://main.net955305.contentfabric.io/config`
working on iphone

**Comment by elv-todd** [comment]
> Hmm, the logic probably should be `if player != bitmovin { all the other logic }`

maybe true, I don't know how to test that.  i want to do no harm.

**Comment by elv-peter** [comment]
I mean you probably don't want to use `playerElement.src` if the player is Bitmovin

File: examples/basic-video-example.html [hunk 1]
```diff
@@ -134,15 +134,15 @@
...
+        value="https://main.net955305.contentfabric.io/config"
         placeholder="Config URL">
       <input
         id="private-key"
         value=""
         placeholder="Private Key">
       <input
         id="content-id"
-        value=""
+        value="hq__FRHfVK9ppcowDmB6s48cZtHFbJ5Qgduq126eLXJBfPbrmkATowwzz9mjzMfns5B5WpHwBfFYfG"
```
Comment (elv-serban) [inline examples/basic-video-example.html:145]: Let's use this dedicated example video: hq__CcdV4wnCNq9wv6jXpYeCQ2GE4FLQBFtVSSSt2XKfBJMrH89DFDGsfkpWWvBy16QBGGYeF5mLGo

**Review by elv-serban — APPROVED** [review]
Looks good

**Review by elv-peter — APPROVED** [review]

**Pull request merged**